astronomer/providers/microsoft/azure/example_dags/example_wasb_sensors.py

```diff
@@ -147,7 +148,10 @@ def delete_blob_callable(container_name: str, blob_name: str) -> None:
         trigger_rule="all_done",
     )
 
+    end = EmptyOperator(task_id="end")
+
     create_data_storage_container >> upload_blob
     upload_blob >> [wasb_blob_sensor, wasb_prefix_sensor]
     [wasb_blob_sensor, wasb_prefix_sensor] >> delete_blob
     delete_blob >> delete_data_storage_container
+    delete_data_storage_container >> end
```

Comment:
all the preceding tasks should be in a list , isnt it?

start >> [task_A, task_B]>> end

Reply:
yes, right. Thanks, made the change